src/components/switch/switch.stories.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from "react";
+import { ComponentStory, StoryFn } from "@storybook/react";
+
+import Switch, { SwitchProps } from ".";
+
+export const Default: ComponentStory<typeof Switch> = (args) => {
+  const [isChecked, setIsChecked] = useState(false);
+  return (
+    <Switch
+      label="Label"
+      name="switch-name"
+      checked={isChecked}
+      onChange={(e) => setIsChecked(e.target.checked)}
+      {...args}
+    />
+  );
+};
```

Comment:
Looking at this, in my opinion I think we should render the full source of the story rather than just the output to render.
I understand your reason against this since its expected our consumers would already be fully familiar with React state, but I feel including the state in the snippet is good for completeness, and would be consistent with how our other components, that are controlled by default, are documented.